tools/controller.js

```diff
@@ -96,7 +96,7 @@ Controller.prototype.onToggleLineBreakpoint = function (line, clickedSpan) {
   this.view.updateLineBreakpoint(breakpoint);
 };
 
-Controller.prototype.onToggleMessageSendBreakpoint = function (e) {
+/*Controller.prototype.onToggleMessageSendBreakpoint = function (e) {
```

Comment:
Why is this code comment out instead of being removed?
Unused code should be removed. If it is later needed again, we can find in git/on github